packages/ozone/src/api/verification/grant.ts

```diff
@@ -0,0 +1,67 @@
+import { AuthRequiredError, InvalidRequestError } from '@atproto/xrpc-server'
+import { AppContext } from '../../context'
+import { Server } from '../../lexicon'
+import { getReposForVerifications } from '../../verification/util'
+
+export default function (server: Server, ctx: AppContext) {
+  server.tools.ozone.verification.grant({
+    auth: ctx.authVerifier.modOrAdminToken,
+    handler: async ({ input, auth, req }) => {
+      if (!ctx.cfg.verifier) {
+        throw new InvalidRequestError('Verifier not configured')
+      }
+
+      if (!auth.credentials.isAdmin && !auth.credentials.isVerifier) {
```

Comment:
If I'm understanding correctly admins have `isVerifier` flipped to `true`, so may be able to simplify the check.

Reply:
All good either way though if this is clearer 👍